internal/service/gamelift/game_session_queue.go

```diff
@@ -36,6 +36,10 @@ func ResourceGameSessionQueue() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringLenBetween(1, 128),
 			},
+			"notification_target": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
lets add arn validation as docs state that its `An SNS topic ARN that is set up to receive game session placement notifications.`

Reply:
I didn't find any helpers for ARN validation but found some examples of validating them using regexes. Is it what you want?